instance/tasks.py

```diff
@@ -60,3 +63,40 @@ def spawn_appserver(instance_ref_id, mark_active_on_success=False, num_attempts=
                 # finish and replace the second as the active server. We are not really worried about that for now.
                 instance.set_appserver_active(appserver_id)
             break
+
+
+@db_task()
+def shut_down_instances():
```

Comment:
Nit: Worth having a more descriptive method name? It makes it sound more generic than it is.